code/datums/map_config.dm

```diff
@@ -61,11 +62,18 @@
 	return configs
 
 #define CHECK_EXISTS(X) if(!istext(json[X])) { log_world("[##X] missing from json!"); return; }
-/datum/map_config/proc/LoadConfig(filename, error_if_missing, maptype)
+/datum/map_config/proc/LoadConfig(filename, error_if_missing, maptype, load_default)
 	if(!fexists(filename))
 		if(error_if_missing)
 			log_world("map_config not found: [filename]")
-		return
+		if(!load_default)
+			return
+		switch(maptype)
+			if(GROUND_MAP)
+				return LoadConfig("_maps/lv624.json", error_if_missing, maptype)
+			if(SHIP_MAP)
+				return LoadConfig("_maps/pillar_of_spring.json", error_if_missing, maptype)
```

Comment:
next_map.json is not on the git repo, so integration test couldn't load it. Added that as a backup